packages/falcon-magento2-api/src/index.js

```diff
@@ -65,6 +64,10 @@ module.exports = class Magento2Api extends Magento2ApiBase {
       },
       PaymentMethod: {
         config: apiGetter((api, ...args) => api.getPaymentMethodConfig(...args))
+      },
+      Address: {
+        country: apiGetter((api, ...args) => api.country(...args)),
+        region: apiGetter((api, ...args) => api.region(...args))
```

Comment:
I would rename it to `getAddressRegion` as this resolver is `Address` specific (we have a similar situation with `Product` and `this.productPrice`)

```suggestion
        region: apiGetter((api, ...args) => api.addressRegion(...args))
```